tests/test_state_props/test_is_separable.py

```diff
@@ -2,7 +2,7 @@
 import numpy as np
 
 from toqito.state_props.is_separable import is_separable
-from toqito.channels import partial_trace
+from picos import partial_trace
```

Comment:
Hmm, I think what we want to do here is the same that you did for the `partial_transpose` function.

Namely, we want to invoke the `partial_trace` function from `toqito` (instead of from `picos`) with the change that the index offset is `0` instead of `1` (as you did for the `partial_transpose`)

Does that make sense?

Reply:
Yeah, I just changed that